projects/admin/src/app/acquisition/api/acq-receipt-api.service.ts

```diff
@@ -17,77 +17,117 @@
  */
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { RecordService } from '@rero/ng-core';
+import { RecordService, RecordUiService } from '@rero/ng-core';
 import { Observable, of } from 'rxjs';
 import { catchError, map } from 'rxjs/operators';
-import { IReceiptOrder } from '../classes/receipt';
-import { IReceiptLine, IReceiptOrderLine, IResponseReceiptLine } from '../components/order/order-receipt-view/order-receipt';
+import { BaseApi } from '../../../../../shared/src/lib/api/base-api';
+import { IAcqReceiptLine } from '../classes/receipt';
+import { IResponseReceiptLine } from '../components/receipt/receipt-form/order-receipt';
 
 @Injectable({
   providedIn: 'root'
 })
 export class AcqReceiptApiService {
 
-  /** Resource name */
-  private _resource = 'acq_receipts';
+  private readonly _defaultRecordOptions = {
+    headers: { Accept: 'application/json'},
```

Comment:
as I used `option.headers` to call `getRecords()` I need to set a default value otherwise an error occurred "'headers' is an unknown property for 'options'"